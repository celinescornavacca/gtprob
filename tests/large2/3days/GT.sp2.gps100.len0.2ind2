((((((((A37,A38),A21),(A9,(((((A77,A78),A69),A42),A36),((A44,A45),A17)))),A1),(A4,A5)),(((A94,A95),((A7,((A79,A80),A66)),((A15,(A92,A93)),(A6,(((A84,A85),(A57,A58)),(A25,A26)))))),((A3,((((A12,((A86,A87),A43)),(A67,A68)),(A13,((A23,A24),(A39,(A50,A51))))),(A46,((A59,A60),(A56,(A70,A71)))))),((A64,A65),((A73,A74),(A27,A28)))))),(((((A82,A83),A35),((A40,A41),A31)),(A22,(((A90,A91),(A55,(A62,A63))),A30))),((((((A32,A33),(A34,((A88,A89),A52))),(A18,(A98,A99))),(((((A16,(A19,A20)),A10),A2),(A14,((A53,A54),A29))),A0)),(((A61,(A75,A76)),(A11,(A48,A49))),A8)),(((A81,(A96,A97)),A72),A47)))),((((((B56,(B61,(B80,B81))),B5),(((((B45,B46),(B78,B79)),((B48,B49),B16)),(((B90,(B94,B95)),(B47,(B50,B51))),B8)),(B34,B35))),((((B65,(B73,(B86,B87))),(B88,B89)),B6),((((B37,(B76,B77)),(B40,B41)),(((B11,((B42,(B71,B72)),B30)),(B96,B97)),((B4,(B24,B25)),(((((B28,B29),(B64,(B84,B85))),(B54,B55)),B17),(B22,B23))))),(B38,(B52,B53))))),((B9,(B18,B19)),B7)),((((B67,B68),(B26,B27)),((B0,B1),(((((B43,B44),((B31,B32),B10)),(B2,((B58,(B69,B70)),B36))),(B15,(B20,((B98,B99),B91)))),((((((B62,B63),(B82,B83)),B39),B33),(B74,B75)),(B14,(B21,(B57,(B66,(B92,B93))))))))),(B3,((B59,B60),(B12,B13))))));
