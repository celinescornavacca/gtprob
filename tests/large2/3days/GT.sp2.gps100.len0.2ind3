(((((((A3,((A56,A57),(A41,A42))),(A8,A9)),((((A60,(((A76,A77),A71),A64)),A35),(A20,A21)),(A74,A75))),((A4,((A10,(A37,((A85,A86),A48))),(A13,(A43,(A53,((A78,(A83,A84)),A69)))))),(A67,A68))),((((A1,(((A51,A52),A46),((A58,A59),((A54,A55),A22)))),((A5,(A7,(((A73,((A92,A93),A87)),(A94,A95)),A16))),(A36,(A44,A45)))),((((A81,A82),A25),((A11,A12),A6)),((A88,A89),A38))),(((A15,(A19,(A27,((A98,A99),A61)))),((((A28,A29),(A49,A50)),A14),(A31,A32))),(A0,(((A30,(A39,A40)),A26),A2))))),(((A24,(A96,A97)),(A23,(A33,A34))),(((A47,(A70,(A79,A80))),(((A62,A63),((A90,A91),A72)),(A17,A18))),(A65,A66)))),(((((B85,B86),(B98,B99)),((((B60,B61),(B50,B51)),(B19,B20)),(B1,((B87,B88),((((B24,((B43,B44),(B45,B46))),(B33,(B93,B94))),B5),(B79,B80)))))),(((B66,B67),B49),((B40,B41),B37))),(((B0,(B13,B14)),(((((B54,B55),B4),((B15,(B21,B22)),((B77,B78),(B64,B65)))),((B7,B8),B6)),((((B47,B48),(((B95,(B96,B97)),B27),(((B71,B72),B57),(B62,B63)))),((B28,B29),B12)),((B83,B84),(B9,B10))))),(((((B58,B59),B3),(B16,(B53,(B81,B82)))),(B25,B26)),((((B2,(B91,B92)),((B38,B39),(((B35,B36),((B30,(B75,B76)),(((B89,B90),B42),B34))),((B31,B32),B23)))),((B17,B18),((B56,(B69,B70)),B52))),((B68,(B73,B74)),B11))))));
