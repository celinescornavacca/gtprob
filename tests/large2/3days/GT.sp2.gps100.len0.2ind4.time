2.48user 0.22system 0:02.95elapsed 91%CPU (0avgtext+0avgdata 53036maxresident)k
137inputs+0outputs (0major+26411minor)pagefaults 0swaps
