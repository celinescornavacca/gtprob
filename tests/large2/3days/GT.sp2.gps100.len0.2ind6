(((((((A92,A93),(A86,A87)),A3),(((((A56,(A90,A91)),(((A39,(A98,A99)),A31),(A10,A11))),((((A70,A71),A38),((A88,A89),A19)),(A76,A77))),(A9,(A15,(A32,(A46,A47))))),((A7,A8),((A16,(A24,A25)),((A36,A37),A28))))),(((((A33,(A57,A58)),((((A30,(A84,A85)),(A80,A81)),((A44,A45),(A13,A14))),((A29,((A96,A97),((((A52,A53),(A94,A95)),A43),A40))),(A82,A83)))),((A65,(A66,A67)),A35)),A2),((A1,((A5,(((A17,A18),((((A78,A79),A34),A23),A12)),(A74,A75))),(((A48,A49),((A61,A62),A42)),((A72,A73),(A54,A55))))),((A41,(A59,A60)),(A4,((A21,(A26,A27)),(A50,A51))))))),((((A20,((A68,A69),A22)),(A63,A64)),A6),A0)),(((((((((B92,B93),B53),(B22,((B32,(B62,B63)),((B78,B79),B34)))),B7),((B29,B30),(B14,((B46,B47),(B86,B87))))),(((B15,B16),B13),(B25,(((B72,(B98,B99)),B38),((((B80,B81),B52),B50),(B39,B40)))))),((((B1,(B27,((B64,(B76,B77)),(B67,(B82,B83))))),((B21,(B70,B71)),(B3,((B10,B11),(B58,B59))))),(B2,(B4,(B12,(B88,B89))))),((((B26,(B65,B66)),((B28,(B35,B36)),B23)),(B17,B18)),((B41,B42),((B96,B97),B20))))),(((((B84,B85),B73),(B68,B69)),(((B54,B55),B51),(B90,B91))),(((B74,B75),B9),((B6,((B31,(B60,B61)),(B48,B49))),(B8,((B94,B95),B19)))))),((((B33,((B43,(B56,B57)),B37)),B5),(B24,(B44,B45))),B0)));
