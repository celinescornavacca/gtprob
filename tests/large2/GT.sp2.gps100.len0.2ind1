((((((A8,A9),(((A41,A42),A4),((((A18,(A30,A31)),((A88,A89),A26)),((A82,A83),A55)),A13))),((A29,(A33,A34)),A11)),((A49,A50),A2)),(((((((A92,A93),A71),A65),((A39,A40),((A96,A97),(A35,(A48,(A66,(A73,A74))))))),(A5,(((A27,A28),A15),A10))),(A0,((A19,A20),(A12,(A94,A95))))),(((((A43,(A69,A70)),A25),(((A86,A87),A56),(A98,A99))),((((A3,((A14,((((A46,A47),(A53,A54)),A32),(A60,A61))),(A57,A58))),(A36,A37)),(A21,A22)),(((A44,(A77,A78)),((A45,((A84,A85),A81)),(A62,(A67,A68)))),((((A23,A24),(((A16,A17),A7),(A51,A52))),A1),((((A75,A76),A72),(A79,A80)),(A6,((A59,(A63,A64)),A38))))))),(A90,A91)))),((((B3,B4),B0),(((B22,(B92,B93)),((B52,B53),B14)),(((((B94,B95),B91),B36),(B45,B46)),(((B96,B97),B73),(B63,B64))))),((((B83,B84),B82),((B39,B40),(((((B25,(B74,B75)),B17),((B11,B12),B9)),B6),B5))),((((B1,((B7,(((B26,B27),((B60,B61),((B87,B88),B47))),((B35,(B66,B67)),B19))),((((B85,B86),(B13,((B80,B81),(B37,B38)))),((B23,B24),((B89,B90),B70))),(((B33,(B78,B79)),B29),(B65,(B98,B99)))))),((((B15,B16),((B68,B69),(B28,((B50,B51),B34)))),(((B31,B32),(B18,(B41,(B56,B57)))),(((B44,(B76,B77)),(B20,B21)),(B54,B55)))),((B48,B49),B10))),(B2,(B8,(B62,(B71,B72))))),((B58,B59),(B30,(B42,B43)))))));
