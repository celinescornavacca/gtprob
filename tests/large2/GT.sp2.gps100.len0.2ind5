((((((((A39,A40),(A70,A71)),(A18,(A31,A32))),((A6,(A74,A75)),((A7,(A19,((A24,(A59,A60)),A20))),A3))),(((((((A87,A88),(A68,A69)),A11),(A28,(A53,(A85,A86)))),A0),(((((A21,((A76,A77),(A64,A65))),(A16,(((A90,A91),(A62,A63)),(((A72,A73),A61),A49)))),(((A12,(A52,((A94,A95),A84))),(A10,(A78,A79))),((A80,A81),A5))),(((A47,A48),A4),((A37,(A92,A93)),A8))),((A9,(A14,A15)),(((A50,A51),(A33,A34)),(A25,A26))))),(((A35,A36),A29),(A23,(A82,A83))))),(A56,A57)),(((A27,(A66,A67)),((A17,(A54,A55)),(((A98,A99),((A45,A46),A38)),A13))),(A1,((((A43,A44),A42),A41),(((A30,(A58,(A89,(A96,A97)))),A22),A2))))),(((((B32,(B66,B67)),(B12,((B41,(B79,B80)),(((B29,(B49,(B51,B52))),B28),(B35,(B85,B86)))))),((B8,((B26,B27),(B33,B34))),(((B75,B76),B63),(B39,(B55,B56))))),((((B37,(B59,B60)),(B96,B97)),B18),B2)),((((B14,((B83,B84),(B68,B69))),((B71,B72),B45)),((((B0,((((B61,B62),B54),((B64,B65),B38)),B1)),((B13,(B20,B21)),B4)),(B57,B58)),((((B6,(B50,(B88,B89))),(B3,((B73,B74),B7))),(B25,((B77,B78),(B46,B47)))),(((B30,B31),(B11,(((B24,(B42,B43)),B15),(B70,(B81,B82))))),(B5,((B87,(B92,B93)),(B48,(B90,B91)))))))),((B44,(B94,B95)),(((B17,((((B98,B99),B53),B40),B36)),(B16,(B19,(B22,B23)))),(B9,B10))))));
