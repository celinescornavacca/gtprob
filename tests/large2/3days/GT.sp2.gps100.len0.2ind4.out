-834.731056188
